src/builtin_complete.cpp

```diff
@@ -193,11 +184,11 @@ int builtin_complete(parser_t &parser, io_streams_t &streams, wchar_t **argv) {
                 break;
             }
             case 'u': {
-                authoritative = 0;
+                streams.err.append_format(L"%ls: -u / --unauthoritative flags have been removed\n", cmd);
```

Comment:
Ahh, I see you were just following the pattern of the surrounding code. I'll fix that when I merge your change.